reactor-core/src/main/java/reactor/core/publisher/Flux.java

```diff
@@ -2100,6 +2101,64 @@ public static <T, D> Flux<T> using(Callable<? extends D> resourceSupplier, Funct
 		return using(resourceSupplier, sourceSupplier, resourceCleanup, true);
 	}
 
+	/**
```

Comment:
For alignment purposes, I'd suggest moving the two new methods below the existing `using` methods but before `usingWhen`.

Reply:
✅ updated!